scripts/dry-run-workloads.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+SCRIPTS_DIR="$(dirname "${BASH_SOURCE[0]}")"
+ROOT_DIR="$(cd "${SCRIPTS_DIR}/.." && pwd)"
+BUILD_DIR="${ROOT_DIR}/build"
+
+CMD=(
+    "${GENNY:-genny}"
+    --dry-run
+)
+
+# Find all of the workload yamls at the specified directory
+WORKLOAD_DIR="${1}"
+WORKLOAD_LIST="${BUILD_DIR}/workload-list.txt"
+find "${WORKLOAD_DIR}" \( -iname "*.yml" -or -iname "*.yaml" \) -print >"${WORKLOAD_LIST}"
```

Comment:
1. Similar to above, the script can know in advance where workload files live I don't know if there's a use-case for wanting to run it against a different directory?
2. we only use the `.yml` suffix so no need to `-or`
3. why cat to a separate file? should be able to just do this "in-memory"?

Reply:
1. Well, if someone decides to maintain their own folder of workloads, this script will still work for them.
2. I mean, we don't enforce the extension as far as I can tell. I'm not sure it's worth enforcing either, yaml is the official extension -- https://yaml.org/faq.html.
3. I tried that. And amazon linux 2 shit the bed. I dunno.